src/test-resources/basic-axe-v3.2.2-sarif-v2.1.2.sarif

```diff
@@ -0,0 +1,768 @@
+{
+    "version": "2.1.2",
+    "runs": [
+      {
+        "conversion": {
+          "tool": {
+            "driver": {
+              "name": "axe-sarif-converter",
+              "fullName": "axe-sarif-converter v1.3.0",
+              "version": "1.3.0",
```

Comment:
We should make sure to add a tracking story to start picking this up from package.json instead of hard-coding it.